_woc/Weekly_blogs.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Text Sentiment Analysis Starter Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
+  - Winter of Code
+  - Developers
+  - Developer Student Clubs
+  - DSC NSEC
+  - DSC IEM
+
+
+last_modified_at: 2020-12-20T08:06:00-05:00
+og_image: /assets/images/WoC2020logo.jpg
+header:
+  teaser: "/assets/images/WoC2020logo.jpg"
```

Comment:
Change the header images to your own favourite image. Make the website stand out and not dull. :)

Reply:
All images go under assets/images . https://github.com/khanfarhan10/khanfarhan10.github.io/tree/master/assets/images